app/models/destination.rb

```diff
@@ -0,0 +1,10 @@
+class Destination < ApplicationRecord
+  has_many :reservations
+  has_many :users, through: :reservations
+
+  validates :name, presence: true, length: { maximum: 100 }
+  validates :location, presence: true, length: { maximum: 100 }
+  validates :image_url, presence: true
+  validates :price_per_day, presence: true, numericality: true
```

Comment:
[Optional]
Instead of using image_url maybe you could use ` carrierwave ` gem for uploading files. I think it will make our frontend app to be faster. This is just optional and we can consider this as an improvement if we have extra time at the end of the project.